src/Contrib/Otlp/OtlpTransportFactoryInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Otlp;
+
+use OpenTelemetry\SDK\Common\Export\TransportFactoryInterface;
+
+interface OtlpTransportFactoryInterface extends TransportFactoryInterface
```

Comment:
IMO this should be internal to `SpanExporterFactory` / I don't think we need this interface. We could resolve the grpc endpoint similar to how we resolve the http endpoint.

Reply:
removed the interface and moved the logic to SpanExporterFactory